packages/react-popover/src/components/Popover/Popover.types.ts

```diff
@@ -103,7 +103,7 @@ export type PopoverState = PopoverCommons &
 /**
  * Data attached to open/close events
  */
-export type OnOpenChangeData = Pick<PopoverState, 'open'>;
+export type OnOpenChangeData = { open: boolean };
```

Comment:
As I mentioned I had to simplify this to avoid handling of `Pick<>` which is quite complex